sh/deploy_new_bridge_settler.sh

```diff
@@ -0,0 +1,299 @@
+    exit 1
+fi
+
+. "$project_root"/sh/common_safe_deployer.sh
+. "$project_root"/sh/common_deploy_bridge_settler.sh
+
+# set minimum gas price to (mostly for Arbitrum and BNB)
+declare -i min_gas_price
+min_gas_price="$(get_config minGasPriceGwei)"
+min_gas_price=$((min_gas_price * 1000000000))
+declare -r -i min_gas_price
+declare -i gas_price
+gas_price="$(cast gas-price --rpc-url "$rpc_url")"
+if (( gas_price < min_gas_price )) ; then
+    echo 'Setting gas price to minimum of '$((min_gas_price / 1000000000))' gwei' >&2
+    gas_price=$min_gas_price
+fi
+declare -r -i gas_price
+declare -i gas_estimate_multiplier
+gas_estimate_multiplier="$(get_config gasMultiplierPercent)"
+declare -r -i gas_estimate_multiplier
+
+while (( ${#deploy_calldatas[@]} >= 3 )) ; do
+    declare -i operation="${deploy_calldatas[0]}"
+    declare deploy_calldata="${deploy_calldatas[1]}"
+    declare target="${deploy_calldatas[2]}"
+    deploy_calldatas=( "${deploy_calldatas[@]:3:$((${#deploy_calldatas[@]}-3))}" )
+
+    declare packed_signatures
+    packed_signatures="$(retrieve_signatures bridge_settler_confirmation "$deploy_calldata" $operation "$target")"
+    packed_signatures="${packed_signatures:2}"
+    declare packed_signatures_length="${#packed_signatures}"
+    packed_signatures_length=$(( packed_signatures_length / 2 ))
+    packed_signatures_length="$(cast to-uint256 $packed_signatures_length)"
+    packed_signatures_length="${packed_signatures_length:2}"
+
+    # pad the deployment call
+    deploy_calldata="${deploy_calldata:2}"
+    declare deploy_calldata_length="${#deploy_calldata}"
+    declare -i deploy_calldata_padding_length=$((deploy_calldata_length % 64))
+    if (( deploy_calldata_padding_length )) ; then
+        deploy_calldata_padding_length=$((64 - deploy_calldata_padding_length))
+        deploy_calldata="$deploy_calldata""$(seq 1 $deploy_calldata_padding_length | xargs printf '0%.0s')"
+    fi
+    deploy_calldata_length=$(( deploy_calldata_length / 2 ))
+    deploy_calldata_length="$(cast to-uint256 $deploy_calldata_length)"
+    deploy_calldata_length="${deploy_calldata_length:2}"
+
+    ## assemble the call to `execTransaction`
+    # we have to do it this awkward way instead of using `cast calldata` because
+    # `$deploy_calldata` is longer than allowed for a command-line argument
+    declare packed_calldata
+    packed_calldata="$(cast concat-hex "$execTransaction_selector" "$(cast to-uint256 "$target")" "$(cast to-uint256 0)" "$(cast to-uint256 320)" "$(cast to-uint256 $operation)" "$(cast to-uint256 0)" "$(cast to-uint256 0)" "$(cast to-uint256 0)" "$(cast to-uint256 "$(cast address-zero)")" "$(cast to-uint256 "$(cast address-zero)")" "$(cast to-uint256 $((320 + 32 + ${#deploy_calldata} / 2)))")""$deploy_calldata_length""$deploy_calldata""$packed_signatures_length""$packed_signatures"
+
+    ## set gas limit and add multiplier/headroom (again mostly for Arbitrum)
```

Comment:
all of the direct manipulation of the RPC using `jq` and `curl` from here down is not needed if you assume that the calldata is not so excessively long that it can't be passed as a command-line argument. of course, if you don't care to simplify this, you might want to reify this pattern so that you're not having tons of code duplication with the existing script

Reply:
Reverted the changes in this commit 2b586d6 where the manipulation was introduced